<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseCreator</class>
 <widget class="QWidget" name="DatabaseCreator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(44, 44, 52);</string>
  </property>
  <widget class="QPushButton" name="openKey">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>210</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
border:0px;
background-color: rgb(68, 158, 241);
color: rgb(255, 255, 255);
font-weight: bold
}

QPushButton::pressed {
background-color: rgb(93, 127, 247);
}</string>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(68, 158, 241);
color: rgb(238, 238, 236);
font: 75 22pt &quot;DejaVu Sans&quot;;
font-weight: bold</string>
   </property>
   <property name="text">
    <string>   Blue</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>70</y>
     <width>431</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 16pt &quot;DejaVu Sans&quot;;
background-color: rgb(20, 20, 20);</string>
   </property>
   <property name="text">
    <string>                    Create database</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>571</width>
     <height>611</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(34, 35, 40);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>360</y>
     <width>571</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(34, 35, 40);</string>
   </property>
   <widget class="QPushButton" name="close">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border:0px;
background-color: rgb(239, 41, 41);
color: rgb(255, 255, 255);
font-weight: bold
}

QPushButton::pressed {
background-color: rgb(164, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ok">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>120</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border:0px;
background-color: rgb(68, 158, 241);
color: rgb(255, 255, 255);
font-weight: bold
}

QPushButton::pressed {
background-color: rgb(93, 127, 247);
}</string>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="open">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>140</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
border:0px;
background-color: rgb(68, 158, 241);
color: rgb(255, 255, 255);
font-weight: bold
}

QPushButton::pressed {
background-color: rgb(93, 127, 247);
}</string>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="database">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>471</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 238, 236);
color: rgb(0, 0, 0);
border-radius:0px</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>10000</number>
   </property>
   <property name="placeholderText">
    <string>Path of the database..</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="key">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>210</y>
     <width>471</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 238, 236);
color: rgb(0, 0, 0);
border-radius:0px</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>10000</number>
   </property>
   <property name="placeholderText">
    <string>Path to key..</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>170</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(34, 35, 40);
color: rgb(255, 255, 255);
font: 8pt &quot;DejaVu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>You can add any file as a key, as long as it is never modified and not empty
It is needed for decryption and has to be provided to open the database.</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="stretchTime">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>250</y>
     <width>101</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox {
background-color: rgb(255, 255, 255);
border-radius:0px
}
QSpinBox::up-button {
background-color: rgb(115, 210, 22);
}

QSpinBox::down-button {
background-color: rgb(239, 41, 41);
}</string>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::UpDownArrows</enum>
   </property>
   <property name="suffix">
    <string> sec</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>999999999</number>
   </property>
   <property name="value">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>250</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(34, 35, 40);
color: rgb(255, 255, 255);
font: 8pt &quot;DejaVu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>Stretch time (time taken to open the database)
Prevents effectiveness of bruteforce attacks </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>290</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(34, 35, 40);
color: rgb(255, 255, 255);
font: 8pt &quot;DejaVu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>Number of iterations made to open the database
Prevents effectiveness of bruteforce attacks
Don't use this parameter if you're not familiar with it</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="iterationTime">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>300</y>
     <width>101</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox {
background-color: rgb(255, 255, 255);
border-radius:0px
}
QSpinBox::up-button {
background-color: rgb(115, 210, 22);
}

QSpinBox::down-button {
background-color: rgb(239, 41, 41);
}</string>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::UpDownArrows</enum>
   </property>
   <property name="suffix">
    <string/>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>999999999</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>276</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(34, 35, 40);
color: rgb(255, 255, 255);
font: 8pt &quot;DejaVu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>or</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>openKey</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>widget</zorder>
  <zorder>open</zorder>
  <zorder>database</zorder>
  <zorder>key</zorder>
  <zorder>label_4</zorder>
  <zorder>stretchTime</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>iterationTime</zorder>
  <zorder>label_7</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
